srcpkgs/oomox/template

```diff
@@ -22,12 +22,12 @@ distfiles="
  https://github.com/nana-4/materia-theme/archive/$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz
  https://github.com/themix-project/oomox-gtk-theme/archive/$_oomox_gtk_theme_ver.tar.gz>oomox-gtk-theme-$_oomox_gtk_theme_ver.tar.gz"
 checksum="
- a32e9159cc772f953ba63caf478271b2152c0daf54db82ae52658d5e18c6e719
- d6e91b9f6ea48c0831eaf8cbf266ad2bac672e02d5d28a91caafcf181956f6fa
- b7bf90cd61f10b8f7afc1442ecde4ba11bdafcb1e48471c81c568d3f6a04f82a
- ebbe38cc62889c58d7b3eb4e21f6c5e2ae54fc16761cf65e99a05571067536ae
- dee2a0b38f8fbca3d2980af89f40309f0cb646fad25b88eab1fae5582047160e
- 98bb87ba18581cd29e1469e6a441f0113d6700e9540492bc4663d0e2d877018b"
+ 20b166fbae2b694897512060256ca47d591dd23ce2d72446d5bcf85ceda4e6db
+ a16e441cbe7c7bf09bdf8fd96b10c314326e3a0fe2e060e06d106d487024cdac
+ 1d1ce5a545864823422936a62c43584e0662d0225dc61a79291bc458d9505b7a
+ 0cecc40173f1ae69af649c630d691934f97da8b0610a4c0388d165e3250ec8d0
+ ee9efc1126350a23fc9a7479cb0b8f732c14a2fc2f6697551038463f47462f36
+ 997e15e2677350989edc8b2c2a8678ceddc23ac231b947fb3afe955445988b2d"
```

Comment:
The new checksums don’t work for me. Did you use `xgensum -i template`?

These are the ones I got:
```
checksum="b6df65d89c41b6cf5992b922f260b850f59fcf1a6d17b27a08fbf36f5ef0fab0
 eb4a5c9edcaca47c22fcb3356268a40fa131c98c489d7fc8f03aff81b1533576
 01d09aaf4079843051c7329d1f01114f71eea11a08a142c27c88026ae1fb6a79
 093929410afd0ebc9108778f60c22f91202e1c85eb098fcfd08cc2d4dab5e490
 4d9bec034767e2a08f843e1dd61bdf7ead97621c2977886a9288fb898342578f
 e54e97e617f05babd3ae9b674073898879af37df4cc623f4d42b94c3053da9fa"
```